plugins/Lv2Effect/Lv2Effect.cpp

```diff
@@ -0,0 +1,110 @@
+/*
+ * Lv2Effect.cpp - implementation of LV2 effect
+ *
+ * Copyright (c) 2018-2019 Johannes Lorenz <j.git$$$lorenz-ho.me, $$$=@>
+ *
+ * This file is part of LMMS - https://lmms.io
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program (see COPYING); if not, write to the
+ * Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA 02110-1301 USA.
+ *
+ */
+
+#include "Lv2Effect.h"
+
+#include <QDebug>
+#include <lv2.h>
+
+#include "AutomatableModel.h"
+#include "Lv2FxControlDialog.h"
+#include "Lv2SubPluginFeatures.h"
+#include "embed.h"
+#include "plugin_export.h"
+
+
+
+
+Plugin::Descriptor PLUGIN_EXPORT lv2effect_plugin_descriptor =
+{
+	STRINGIFY(PLUGIN_NAME),
+	"LV2",
+	QT_TRANSLATE_NOOP("Lv2Effect",
+		"plugin for using arbitrary LV2-effects inside LMMS."),
+	"Johannes Lorenz <j.git$$$lorenz-ho.me, $$$=@>",
+	0x0100,
+	Plugin::Effect,
+	new PluginPixmapLoader("logo"),
+	nullptr,
+	new Lv2SubPluginFeatures(Plugin::Effect)
+};
+
+
+
+
+Lv2Effect::Lv2Effect(Model* parent, const Descriptor::SubPluginFeatures::Key *key) :
+	Effect(&lv2effect_plugin_descriptor, parent, key),
+	m_controls(this, key->attributes["uri"])
+{
+}
+
+
+
+
+Lv2Effect::~Lv2Effect()
+{
+}
+
+
+
+
+bool Lv2Effect::processAudioBuffer(sampleFrame *buf, const fpp_t frames)
+{
+	if(!isEnabled() || !isRunning())
```

Comment:
Add a space after if
```suggestion
	if (!isEnabled() || !isRunning())
```